package.json

```diff
@@ -139,13 +139,14 @@
 		"@types/strip-ansi": "3.0.0",
 		"chai": "4.1.2",
 		"chai-as-promised": "7.1.1",
-		"coveralls": "3.0.0",
+		"coveralls": "3.0.2",
 		"globby": "8.0.1",
 		"husky": "0.14.3",
 		"lint-staged": "5.0.0",
 		"mocha": "4.0.1",
 		"nyc": "11.3.0",
 		"prettier": "1.9.2",
+		"rxjs-compat": "6.3.3",
```

Comment:
is this being used?

Reply:
This is for `husky`, without this it seems like it's not working.
I have no idea why it doesn't work though...